media/docs/build/building_with_sycl_support.md

```diff
@@ -67,20 +67,14 @@ $ CC=clang CXX=clang++ cmake .. -G Ninja \
   -DCMAKE_CXX_FLAGS="-ftemplate-backtrace-limit=0 -fdiagnostics-color=always"  
 ```
 
-Note: When building with SYCL support, CMake currently sets the following environment variable to work around an IGC bug:
-```
-$ export IGC_allowDecompose2DBlockFuncs=0
-```
-
 CMake will check that DPC++ compiler is available in the system,
 and it will download the MKL library if it cannot find it.
 
```

Comment:
```suggestion
To get better performance result we require the following combinations of the environment variables flags to provide better performance hints for generating optimised code.
```bash
 export SYCL_PROGRAM_COMPILE_OPTIONS="-ze-opt-large-register-file -gline-tables-only" 
export ONEAPI_DEVICE_SELECTOR=level_zero:gpu
export IGC_VISAOptions="-perfmodel"
export IGC_VectorAliasBBThreshold=10000
export IGC_ExtraOCLOptions="-cl-intel-256-GRF-per-thread"  
```

Reply:
- `-gline-tables-only` isn't for performance
- we should mention that for AOT those options have to be set during compilation and for JIT when running